README.rst

```diff
@@ -186,17 +192,16 @@ If you want to help developing, run:
     cd skorch
     conda env create
     source activate skorch
-    # install pytorch version for your system (see below)
-    conda install -c conda-forge --file requirements-dev.txt
-    python setup.py develop
+    pip install -r requirements-dev.txt
```

Comment:
Should we have a `environmental-dev.yml` for using conda for development?

Reply:
We could consider this, the only problem is that one of the requirements is not available through conda.

https://github.com/skorch-dev/skorch/pull/607#issuecomment-612365883

Reply:
Its possible to include [pip only packages on the yaml](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually).

Here is an [concrete example that dask-ml uses](https://github.com/dask/dask-ml/blob/5d24b72586b03d6ffe5080f4031c63a5b85954b9/ci/environment-docs.yaml#L46-L49).

Reply:
Now I get it, you would like to have a conda environment file that contains all the dependencies, including for development (not distinct ones like `requirements.txt` and `requirements-dev.txt` whose libarries don't overlap). This could certainly be done.

The question is, do we even need a non-development `environment.yml` or is that only interesting for developers anyway?

Reply:
> The question is, do we even need a non-development environment.yml or is that only interesting for developers anyway?

I agree, `environment.yml` seems only interesting for developers.

As for this PR, using pip to install `requirements-dev.txt` is nicer from a maintenance point of view. (We would not need to keep a `environment.yml` in sycn with the `requirements-dev.txt`.

Reply:
:+1: